src/index.html

```diff
@@ -5,12 +5,51 @@
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+      >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Search bar airbnb</title>
+    <title>Searchbar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+    class="searchForm"
+    action="http://localhost:3000.api" 
+    method="post"
+    data-qa="big"
+    >
+      <label>
+        <img class="lens" 
+        src="/images/Search.svg" 
+        alt="search"
+        >
+        <input 
+        class="searchBar"
+        type="text"
+        name="case1" 
+        placeholder="Try &quot Los Angeles &quot" 
+        data-qa="keypress"
+        >
+      </label>
+    </form>
+
+    <form 
+    class="searchForm"
+    action="http://localhost:3000.api" 
+    method="post"
+    data-qa="small"
+    >
+      <label>
+        <img class="lens-small" 
+        src="/images/Search.svg" 
+        alt="search"
+        >
+        <input 
+        class="searchBar bar-2"
+        type="text" 
+        name="case2"
```

Comment:
use meaningful names 
```suggestion
        name="smallSearch"
```